spinalcordtoolbox/scripts/sct_crop_image.py

```diff
@@ -50,6 +54,16 @@ def get_parser():
         help="Output image. By default, the suffix '_crop' will be added to the input image.",
         metavar=Metavar.str,
     )
+    optional.add_argument(
+        '-dilate',
+        type=list_type('x', int),
+        help="Number of extra voxels to keep around the bounding box on each side. Can be specified as a single "
+             "number (for example, '-dilate 5' to keep a margin of 5 voxels in each direction) or a list of 3 "
+             "numbers separated by 'x' (for example, '-dilate 2x3x0' to keep a margin of 2 voxels on each side in "
+             "the x-axis, 3 voxels on each side in the y-axis, and no extra margin in the z-axis).",
+        default="0",
```

Comment:
Good point, I'd forgotten that we can just _not_ set a default value. (For the record, I had set `"0"` as a string, because `0`-the-int is not a list, and `[0]` would have appeared in the help string as something which can't actually be parsed by `list_type('x', int)`.)